test/functional/glance_controller_test.go

```diff
@@ -158,6 +165,10 @@ var _ = Describe("Glance controller", func() {
 		})
 		It("Should fail if db-sync job fails when DB is Created", func() {
 			mariadb.SimulateMariaDBDatabaseCompleted(glanceTest.Instance)
+			mariadb.SimulateMariaDBAccountCompleted(types.NamespacedName{
```

Comment:
in `glance_test_data.go` we can have something like:

```golang
GlanceDatabaseAccount : types.NamespacedName{
	Namespace: glanceName.Namespace,
	Name:      GlanceServiceUser,
},
```

and we reference it as `glanceTest.GlanceDatabaseAccount` in any `mariadb.SimulateMariaDBAccountCompleted` call.

Reply:
yup, will do

Reply:
so yeah I had it as a namespaced name originally but it seemed to be causing tests to fail, let me look more closely why that was happening

Reply:
it does this:

```
     <*errors.StatusError | 0xc0007446e0>: 
      admission webhook "mglance.kb.io" denied the request: json: cannot unmarshal object into Go struct field GlanceSpec.spec.databaseAccount of type string
      {
          ErrStatus: {
              TypeMeta: {Kind: "Status", APIVersion: "v1"},
              ListMeta: {
                  SelfLink: "",
                  ResourceVersion: "",
                  Continue: "",
                  RemainingItemCount: nil,
              },
              Status: "Failure",
              Message: "admission webhook \"mglance.kb.io\" denied the request: json: cannot unmarshal object into Go struct field GlanceSpec.spec.databaseAccount of type string",
              Reason: "",
              Details: nil,
              Code: 400,
          },
      }

```

so i am trying to figure out where that is happening, what's mapping this to that string field

Reply:
OK. looks like, "dont use that name" is the solution, keep it separate.  got it

Reply:
oh nm, base_test is using it.    sorry for the chatter